graspologic/embed/mase.py

```diff
@@ -125,6 +134,7 @@ def __init__(
             n_iter=n_iter,
             diag_aug=diag_aug,
             concat=concat,
+            n_jobs=n_jobs,
         )
         self.scaled = scaled
 
```

Comment:
```suggestion
        self.n_jobs = n_jobs
```

Reply:
Thanks for reviews @j1c will push a new commit by tommorrow

Reply:
@j1c changed committed and codes are reformatted as per latest commit